File: BracketMap.Web/src/app/models/tournament.ts
import { TournamentEnum } from './tournament.enum';
import { Fight } from './fight';
import { Team } from './team';

export class Tournament {
  id = 0;
  name: string | null = null;
  playersPerTeam: number | null = 1;
  teamsPerFight: number | null = 2;
  status = TournamentEnum.Pending;
  fights: Fight[] = [];
  teams: Team[] = [];
}

File: BracketMap.DAL/Entities/Tournament.cs
using System.Collections.Generic;

namespace BracketMap.DAL.Entities
{
    public class Tournament
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int PlayersPerTeam { get; set; }
        public int TeamsPerFight { get; set; }
        public string Status { get; set; }
        //public ICollection<Fight> Fights { get; set; }
        //public ICollection<Team> Teams { get; set; }
    }
}
